products/BellHybrid/services/evtmgr/EventManager.cpp

```diff
@@ -173,8 +173,8 @@ void EventManager::buildKeySequences()
 
     auto resetSeq      = std::make_unique<ResetSequence>(*this);
     resetSeq->onAction = [this]() {
-        app::manager::Controller::sendAction(
```

Comment:
I'm not sure what's the current status of this popup (whether it is shown at all), but isn't this needed?

Reply:
This popup is not used yet because the timeout for this is ~10s. However, manual reset takes 6,7s. Probably the ecoboot will be responsible for displaying this popup, but there is no final decision so I decided to leave this popup in the codebase.

Reply:
Alright, thanks for clarification!